questions.md

```diff
@@ -0,0 +1,2 @@
+How to pull a specific branch into another folder?
```

Comment:
@ayazhassan I'm not sure I understand this  question? Could you elaborate about what you mean?